source/reference/operator/aggregation/and.txt

```diff
@@ -66,10 +63,27 @@ evaluation after encountering the first ``false`` expression.
 
      - ``false``
 
+Error Handling
+--------------
```

Comment:
Hey Jason, sorry for not being clearer in my comment. I meant to make these sub-headings of the "Behavior" section above.